tycho-core/src/test/java/org/eclipse/tycho/core/ee/StandardExecutionEnvironmentTest.java

```diff
@@ -61,8 +60,6 @@ public void setUp() throws Exception {
                 new SilentLog());
         javaSE17Environment = ExecutionEnvironmentUtils.getExecutionEnvironment("JavaSE-17", null, null,
                 new SilentLog());
-        javaSE16Environment = ExecutionEnvironmentUtils.getExecutionEnvironment("JavaSE-16", null, null,
```

Comment:
Maybe instead of removing old version we can have 
`ExecutionEnvironmentUtils.getUnsupportedExecutionEnvironment("JavaSE-16");`
method that prints out an error message "<profilename> is no longer supported" or something?

Reply:
Let's not complicate things. Dead is dead.